_docs/floodgate/setup.md

```diff
@@ -12,17 +12,19 @@ Geyser: [https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/](https://
 ## Prerequisites
 
 - You must own/manage the server in order to add Floodgate. *If you do not own the server, Floodgate does not allow you to bypass logging into Java servers that require a Mojang Java account*.
-- You must either be running Geyser as a [plugin](/geyser/setup/), or be running [Geyser Standalone](/geyser/setup/#standalone-setup). Floodgate does not replace Geyser.
+- You must either be running Geyser as a [plugin](/geyser/setup/), or be running [Geyser Standalone](/geyser/standalone/). Floodgate does not replace Geyser.
 - `floodgate-spigot.jar` cannot be installed on CraftBukkit/Bukkit servers.
+- `floodgate-fabric.jar` can be downloaded here: [floodgate-fabric downlaod](https://ci.opencollab.dev/job/GeyserMC/job/Floodgate-Fabric/job/master/)
 
 ## Setting Up
-*Any reference to Spigot here also refers to any compatible server software such as Paper.*
+*Any reference to Spigot here also refers to any compatible server software such as Paper. For the proxy setup, backend server implies all the servers that are behind the proxy.*
 
 For BungeeCord/Velocity setups: you only are required to install Floodgate on the BungeeCord or Velocity proxy unless you want to use the Floodgate API on the backend servers - see [below](#installing-floodgate-also-on-spigot-servers-behind-bungeecord-or-velocity) for the installation process.  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; *Note:* Installing Floodgate on the backend servers will allow Bedrock player skins to display without the Bedrock player having to switch backend servers.
 
 - Download the Floodgate plugin from [here](https://ci.opencollab.dev/job/GeyserMC/job/Floodgate/job/master/) and add it to your plugins folder on your frontend server.
   - `floodgate-spigot.jar` for Spigot, Paper, etc
+  - `floodgate-fabric.jar` for Fabric (requires FabricAPI)
```

Comment:
Can we link to https://www.curseforge.com/minecraft/mc-mods/fabric-api

Reply:
Also the download link for floodgate-fabric should probably be moved here, rather than being in the prerequisite sections

Reply:
good idea, will change in a sec

Reply:
changed, is that better?

Reply:
yee

Reply:
haw